app/controllers/rater_controller.rb

```diff
@@ -4,6 +4,7 @@ def create
     if user_signed_in?
       @obj = params[:klass].classify.constantize.find(params[:id])
       @obj.rate params[:score].to_f, current_user, params[:dimension]
+      @obj.create_activity :rate, owner: current_user, recipient: @obj.try(:user)
```

Comment:
Consider moving `create_activity ...` call on line 7 into the `rate` method called on line 6. One reason you may want to do this is if you'd want every call to `.rate ...` to record an `activity`.